cmd/feegrant.go

```diff
@@ -190,7 +205,7 @@ func feegrantBasicGrantsCmd(a *appState) *cobra.Command {
 
 			granterAcc, err := prov.AccountFromKeyOrAddress(keyNameOrAddress)
 			if err != nil {
-				fmt.Printf("Error retrieving account from key '%s'\n", keyNameOrAddress)
+				a.log.Error("unknown account", zap.String("key or address", keyNameOrAddress))
```

Comment:
nit: logs should begin with an uppercase letter.

Also, should we log the returned error here as well?